src/vercel.ts

```diff
@@ -653,6 +653,73 @@ const completionSpec: Fig.Spec = {
       name: "whoami",
       description: "Shows the username of the currently logged in user",
     },
+    {
+      name: "alias",
+      description:
+        "Apply custom domains based on git branches, or other heuristics",
+      subcommands: [
+        {
+          name: "set",
+          description: "Assign a custom domain to a deployment",
+          args: [
+            {
+              name: "deployment url",
```

Comment:
The name of the argument should be one word if possible, and then you can add a `description` to explain a bit more about it. Can you change this for each argument underneath?

As example:
```suggestion
              name: "url",
              description: "The URL of the deployment to assign to a domain",
```

Reply:
I wonder if we can set up a linter for this? cc @fedeci

Reply:
That would be a good idea!